File: getparts.py
import regex as re

file=open("annotated-6A.txt","r")
#print(file.read())
pattern = r'\*+[A-Z]+[A-Z].*?\#'
txt = re.findall(pattern, file.read())
#print(txt)
for x in txt:
  print(" ".join(map(str,x.split()[1:-1])))

File: GetParts.hs
import Control.Monad
import Text.Megaparsec
import Text.Megaparsec.Char
import Replace.Megaparsec
import Data.Either
import Data.Maybe
import Data.Void

-- main = do
--   let input = "annotated-6A.txt"
--   contents <- readFile input
--   print "contents"
--   print contents
--   print "split"
--   print $ splitCap starHash contents

-- -- find just cn, qcn, np etc.
starHash :: Parsec Void String String
starHash = string "*" >> manyTill anySingle $ string "#"